app/src/main/java/com/erya/application/ButtonsCell.java

```diff
@@ -0,0 +1,19 @@
+package com.erya.application;
+
+import androidx.annotation.NonNull;
+
+public class ButtonsCell extends AbsCell {
+    ButtonsCell(@NonNull ViewType viewType) {
+        super(viewType.BUTTONS);
+    }
```

Comment:
We don't need any arguments here. This should look like
```suggestion
    ButtonsCell() {
        super(ViewType.BUTTONS);
    }
```